TfsMigrate.Core/WorkItemTracking/WorkItemRelationExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static class WorkItemRelationExtensions
         /// <remarks>
         /// The related ID isn't available directly but has to be determined by parsing the URL. Only work items are supported right now.
         /// </remarks>
-        public static int GetRelatedId ( this WorkItemRelation source )
+        public static int GetRelatedId(this WorkItemRelation source)

```

Comment:
Please do not adjust files based upon your own styles. Follow the styling of the existing code.